Competition/src/main/cpp/Lift.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Lift.h"
+
+Lift::Lift() : liftMtrLeft{LiftConstants::VICTOR_ID_LEFT_Lift}, 
```

Comment:
Call the ValorSubsystem constructor in the intializer list. See other subsystems for an example